guides/source/active_record_postgresql.md

```diff
@@ -543,6 +544,32 @@ When building a model with a foreign key that will reference this UUID, treat
 $ rails generate model Case device_id:uuid
 ```
 
+Indexing
+--------
+
+PostgreSQL includes a variety of index options. The following options are
+supported by the PostgreSQL adapter in addition to the
+[common index options](https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_index)
+
+### Include
```

Comment:
Is there a relevant Postgres doc on this? Similar to how other sections have links to the postgres guide, eg line 598.

Reply:
@ghiculescu there is. I added it to the higher level `Indexing` section since the doc covers all index creation and there is no anchor link specific to the `INCLUDE` param.